libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -644,19 +644,10 @@ bool AP_Follow::handle_global_position_int_message(const mavlink_message_t &msg)
     }
     
     // convert global location to local NED frame position
-    if (!_target_location.get_vector_from_origin_NEU(_target_pos_ned_m)) {
+    Vector3p target_pos_ned_cm;
+    if (!target_location.get_vector_from_origin_NEU(target_pos_ned_cm)) {
```

Comment:
could you not use get_vector_from_origin_NEU_m()?

Reply:
Nice